jcl/src/java.base/share/classes/com/ibm/tools/attach/target/IPC.java

```diff
@@ -81,6 +81,22 @@
 	 */
 	public static boolean isWindows = false;
 
+	/**
+	 * True if operating system is z/OS.
+	 */
+	public static boolean isZOS = false;
+
+	static {
+		String osName = com.ibm.oti.vm.VM.getVMLangAccess().internalGetProperties().getProperty("os.name"); //$NON-NLS-1$
+		if (null != osName) {
+			if (osName.equalsIgnoreCase("z/OS")) { //$NON-NLS-1$
+				IPC.isZOS = true;
```

Comment:
There is no need to specify current class `IPC`, similar for `IPC.isWindows` below.